swan-lake/development-tutorials/ballerina-persist/supported-data-stores.md

```diff
@@ -221,3 +222,72 @@ Follow the steps below to obtain API tokens for Google Sheets.
+
+The Redis data store is a key-value-based inmemory datastore useful for storing and retrieving unstructured data quickly. It is not the default data store for `bal persist`. Therefore, you need to explicitly specify the data store when initializing `bal persist` in your application as follows.
+
+```shell
+$ bal persist init --datastore redis
+```
+
+### Supported Ballerina types
+
+The Redis data store supports the following Ballerina types. As we have utilized `hash` and `set` as Redis data types to store objects, the following data types will be converted to `string`.
+
+- int
+- float
+- decimal
+- string
+- time:Date
+- time:TimeOfDay
+- time:Utc
+- time:Civil
+- enum
+
+### Configuration
+
+You need to set values for the following basic configuration parameters in the `Config.toml` file in your project to use the Redis data store.
+
+| Parameter  |             Description              |
+|:----------:|:------------------------------------:|
+| connection | The connection URI of the DB server. |
+
+The following is a sample `Config.toml` file with the Redis data store configuration. This is generated by the `persist generate` command.
+
+```toml
+[<packageName>.<moduleName>.redis]
+connection = "redis://localhost:6379"
+```
+
+Alternatively, you can provide connection parameters instead of the `URI` as follows.
+
+```toml
+[<packageName>.<moduleName>.redis.connection]
+host = "localhost"
+port = 6379
+```
+
+Additionally, you can set values for the advanced configuration parameters in the `Config.toml` file in your project to use the Redis data store. For more information on these parameters, see the [Redis Connector documentation](https://central.ballerina.io/ballerinax/redis/latest#ConnectionConfig).
+
+### Setup guide
+
+#### Setup a Redis server instance
+
+Select one of the methods below to set up a Redis server.
+
+- Install a Redis server on your machine locally by downloading and installing it based on your development platform. See the [official Redis documentation](https://redis.io/download/).
+- Use Docker to create a DB server deployment.
+   1. Install Docker on your machine if you haven't already.
+   2. Pull the Redis Docker image from Docker Hub using the command `docker pull redis`.
+   3. Run the Redis container as follows `docker run -d -p 6379:6379 --name <container-name> redis`.
+- Use a cloud-based DB solution such as Google’s Cloud, Amazon’s Web Services, or Microsoft’s Azure database.
+   1. Visit [Redis cloud console](https://app.redislabs.com).
+   2. Log in using email and password or using one of the Single Sign-On options.
+   3. Choose either Amazon Web Services, Google Cloud, or Microsoft Azure as the database provider.
+   4. Select a region and create the database.
+   5. Find your `username`, `password` and the `public endpoint`
+   6. Replace the `connection` parameter in `Config.toml` file as below
+
+      ```toml
+      connection = "redis://<username>:<password>@<public_endpoint>"
```

Comment:
Missing closing backticks?

Reply:
Closing back-ticks are not necessary since it's the end of the page, however, I have added closing back-ticks here https://github.com/ballerina-platform/ballerina-dev-website/pull/8838/commits/01a0fbe02d78635abd79c12db5ec3562254a9bc3